import.go

```diff
@@ -65,9 +72,15 @@ func (c *Client) GetImport(ctx context.Context, id string) (*GetImportResponse,
 // Note: Do not use this.
 // It is present for internal usage only.
 // This function can, and will, break and/or be removed at any point in time.
-func (c *Client) ListImports(ctx context.Context) (*ListImportsResponse, error) {
+func (c *Client) ListImports(ctx context.Context, opts *ListImportsOptions) (*ListImportsResponse, error) {
```

Comment:
it doesn't matter here but in general, we should have options pattern as a habit/consistency

`ListImports(ctx, WithLimit(10)) // offset is zero` 
`ListImports(ctx, WithLimit(20), WithOffset(10))`